src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPool.cs

```diff
@@ -773,12 +776,7 @@ private void CheckForHttp2ConnectionInjection()
                 _pendingHttp2Connection = true;
 
                 RequestQueue<Http2Connection?>.QueueItem queueItem = _http2RequestQueue.PeekNextRequestForConnectionAttempt();
-
-                // Queue the creation of the connection to escape the held lock
-                ThreadPool.QueueUserWorkItem(static state =>
-                {
-                    _ = state.thisRef.AddHttp2ConnectionAsync(state.queueItem); // ignore returned task
-                }, (thisRef: this, queueItem), preferLocal: true);
```

Comment:
Does this mean the work is more likely to happen on the global queue?

Reply:
No.

Reply:
@HypsyNZ, you've been posting and deleting a bunch of comments that are dismissive or rude, eg
"The amount of production software this is going to break is hilarious." 
Please either stop or be more constructive in your feedback. Thank you.

Reply:
Oh you would rather dismissive and rude comments were public so you could complain about it publicly instead of taking responsibility for the fact that this sudden and unexpected change is going to break software?

Reply:
Yes, I am aware you can read comments that were deleted.

Reply:
> Oh you would rather dismissive and rude comments were public so you could complain about it publicly instead of taking responsibility for the fact that this sudden and unexpected change is going to break software?

This addition has zero impact on existing behavior. What exactly is it going to break?

And we have a code of conduct covering this repo. I ask that you respect it.

Reply:
I'm pretty sure you are breaking your own code of conduct right now but whatever.

Reply:
> Yes, I am aware you can read comments that were deleted. This post is very professional.

I'm subscribed to the repo, and like anyone subscribed, I get notified of all comments, so my inbox has been going up with your unhelpful remarks and borderline obscene snipes. Please stop.

Reply:
Do you realize how much `ConfigureAwait(false)` gets called?

Not going to affect existing code, lmao.

Reply:
> Do you realize how much ConfigureAwait(false) gets called?

Yes, I do.  How exactly do you believe this PR negatively impacts the behavior of `ConfigureAwait(false)` such that it would break anyone?

Reply:
You are just going to argue in bad faith no matter what I say. You already know the impact, You did it.

Reply:
And sorry to everyone else I didn't realize my comments got delivered to everyone geez that is embarassing.

Reply:
> You are just going to argue in bad faith no matter what I say. You already know the impact, You did it.

I honestly have no idea what you're talking about.  This PR adds a new overload that provides _additional_ functionality.  It does not change the existing functionality.  All code using `ConfigureAwait` today continues operating exactly as it did before this change.  Please elaborate on exactly what are your specific concerns for breaking.

Reply:
Also, You posted every single comment so why would you say that there are worse ones?

What you are doing right now breaks Microsofts code of conduct, not just your own.

Reply:
I did not want this interaction in a public setting, You are unprofessional and angry about being called out.

Reply:
Where does it say you are allowed to be deliberately confrontational?

Reply:
What are you talking about which comment would that be?

Reply:
Anyway, The point is, You already know what you have done, There is no point discussing it.

Reply:
@HypsyNZ Just chill and go for a walk, man 🙂

Reply:
> I did not want this interaction in a public setting,

If your don't want to discuss publicly, then don't post your comments publicly.

> You are unprofessional and angry about being called out.

I'm not angry about being called out. I'm legitimately asking what you believe the problem is. If there's actually a functional problem, I want to fix it.

What I'm upset about is your posting on multiple PRs today with inappropriate comments. Multiple people have pinged me about it, concerned about your behavior.

> Anyway, The point is, You already know what you have done, There is no point discussing it.

No, I don't. Please tell me so we can discuss it and fix it is there actually a problem.

Reply:
> @HypsyNZ Just chill and go for a walk, man 🙂

This was marked as resolved yesterday, this whole conversation was unprovoked.

Reply:
If you wanted to discuss my comments then this review is not the place you are breaking the code of conduct.

Reply:
> If you wanted to discuss my comments then this review is not the place

You left a bunch of your comments on this PR, so I'm responding to them on this PR.

Reply:
I have no doubt in my mind you know exactly what you have done, hence the comments that I thought only you could see.

Reply:
> I have no doubt in my mind you know exactly what you have done, hence the comments that I thought only you could see.

For the last time, I do not know what functional problem you're alluding to. Please share the specific concern.

Reply:
Regardless of my comments what you are doing in this review is an embarrassment.

Reply:
> Regardless of my comments what you are doing in this review is an embarrassment.

I'm responding to someone raising concerns and calling them out on verbal abuse throughout the repo today. I'm not embarrassed.

Reply:
You should be.